tests/foreman/ui/test_errata.py

```diff
@@ -757,3 +762,97 @@ def test_positive_filtered_errata_status_installable_param(session, errata_statu
                 if key in expected_values
             }
             assert actual_values == expected_values
+
+
+@tier3
+def test_content_host_errata_search_commands(session, module_org, module_repos_col):
+    """ View a list of affected content hosts for security (RHSA) and bugfix (RHBA) errata,
+    filtered with errata status and applicable flags. Applicability is calculated using the
+    Library, but Installability is calculated using the attached CV, and is subject to the
+    CV's own filtering.
+
+:id: 45114f8e-0fc8-4c7c-85e0-f9b613530dac
+
```

Comment:
we should probably mention the BZ in the docstring

Reply:
oh yes, will do

Reply:
Hello @pondrejk 
I can see :BZ: 1707335 at the end of the docstring. Is that the wrong place?

Reply:
@pondrejk I can see it at the end of the docstring, is that incorrect place?

Reply:
Searching other bugs I see :BZ: typically comes after ID or expectedresults, but sometimes before and sometimes after after CaseLevel or CaseImportance (in the later case, that is often at the end).

Reply:
I think the order of items in docstring is not significant

Reply:
@swadeley Peter is correct, order of metadata doesn't matter

Reply:
In that case, there is nothing wrong here, BZ: 1707335  is present in docstring.